--- regexx-0.98.1.orig/src/regexx.hh
+++ regexx-0.98.1/src/regexx.hh
@@ -34,6 +34,7 @@
 #include <split.hh>
 
 namespace regexx {
+  using namespace std;
 
   /** Class to store atoms.
    *
@@ -566,7 +567,7 @@
   }
 
   inline const std::string&
-  Regexx::replacef(std::string (*_func)(const RegexxMatch&), int _flags = 0)
+  Regexx::replacef(std::string (*_func)(const RegexxMatch&), int _flags)
     throw(CompileException)
   {
     exec(_flags&~nomatch);
@@ -580,7 +581,7 @@
   inline const std::string&
   Regexx::replacef(const std::string& _expr,
 		   std::string (*_func)(const RegexxMatch&),
-		   int _flags = 0)
+		   int _flags)
     throw(CompileException)
   {
     expr(_expr);
@@ -590,7 +591,7 @@
   inline const std::string&
   Regexx::replacef(const std::string& _str, const std::string& _expr,
 		   std::string (*_func)(const RegexxMatch&),
-		   int _flags = 0)
+		   int _flags)
     throw(CompileException)
   {
     str(_str);

diff -urN regexx-0.98.org/Makefile.am regexx-0.98/Makefile.am
--- regexx-0.98.org/Makefile.am	Tue May  6 14:07:02 2003
+++ regexx-0.98/Makefile.am	Tue May  6 14:08:44 2003
@@ -4,5 +4,5 @@
 ## $Date$
 ##
 
-SUBDIRS = @subdirs@ src examples
+SUBDIRS = src examples
 EXTRA_DIST = regexx.spec.in
diff -urN regexx-0.98.org/configure.in regexx-0.98/configure.in
--- regexx-0.98.org/configure.in	Tue May  6 14:07:02 2003
+++ regexx-0.98/configure.in	Tue May  6 14:07:10 2003
@@ -23,7 +23,5 @@
 
 dnl Checks for library functions.
 
-AC_CONFIG_SUBDIRS(pcre)
-
 AC_OUTPUT([Makefile src/Makefile examples/Makefile regexx.spec])
 
diff -urN regexx-0.98.org/src/Makefile.am regexx-0.98/src/Makefile.am
--- regexx-0.98.org/src/Makefile.am	Tue May  6 14:07:02 2003
+++ regexx-0.98/src/Makefile.am	Tue May  6 14:07:34 2003
@@ -6,5 +6,6 @@
 
 lib_LTLIBRARIES = libregexx.la
 libregexx_la_SOURCES = regexx.cc regexx.hh split.cc split.hh
-libregexx_la_LDFLAGS = -version-info 2:0:1 ../pcre/maketables.lo ../pcre/get.lo ../pcre/study.lo ../pcre/pcre.lo
+libregexx_la_LDFLAGS = -version-info 2:0:1
+libregexx_la_LIBADD = -lpcre
 include_HEADERS = regexx.hh split.hh
diff -urN regexx-0.98.org/src/regexx.cc regexx-0.98/src/regexx.cc
--- regexx-0.98.org/src/regexx.cc	Tue May  6 14:07:02 2003
+++ regexx-0.98/src/regexx.cc	Tue May  6 14:07:46 2003
@@ -27,7 +27,7 @@
 // $Date$
 
 #include "regexx.hh"
-#include "../pcre/pcre.h"
+#include <pcre.h>
 
 const unsigned int&
 regexx::Regexx::exec(int _flags)
diff -urN regexx-0.98.org/src/regexx.hh regexx-0.98/src/regexx.hh
--- regexx-0.98.org/src/regexx.hh	Tue May  6 14:23:48 2003
+++ regexx-0.98/src/regexx.hh	Tue May  6 14:24:15 2003
@@ -32,6 +32,7 @@
 #include <string>
 #include <vector>
 #include <split.hh>
+#include <pcre.h>
 
 namespace regexx {
   using namespace std;
@@ -502,8 +503,8 @@
     unsigned int m_matches;
     std::string m_replaced;
 
-    void* m_preg;
-    void* m_extra;
+    pcre* m_preg;
+    pcre_extra* m_extra;
 
   };
 
